lib/src/web/rtc_rtp_transceiver_impl.dart

```diff
@@ -45,15 +46,18 @@ class RTCRtpTransceiverInitWeb extends RTCRtpTransceiverInit {
         if (sendEncodings != null)
           'sendEncodings': sendEncodings!.map((e) => e.toMap()).toList(),
       };
+}
 
-  Map<String, dynamic> toJSMap() => {
+extension RTCRtpTransceiverInitWebExt on RTCRtpTransceiverInit {
+  //
```

Comment:
I think the comment mark can be removed to keep the code tidy.

Reply:
Sure!

Reply:
I did a test on android devices, and I can open simulcast with a soft encoder on the android simulator, but I cannot open simulcast with a Qualcomm Hardware encoder on Samsung S20.

Reply:
It can work normally on ios/mac devices.

Reply:
@cloudwebrtc Thanks for testing! Don't know why I can't get it working, do you have a piece of sample code?

Reply:
https://github.com/pion/ion-sdk-flutter/blob/master/example/lib/views/echotest_view.dart

You can use this code with the latest ion-sfu to use grpc-web signaling, but it will only work on the android emulator.